### DISCUSSION TITLE
How to Full width content

### DISCUSSION THREAD

**Discussion by skoenfaelt** [nuxt-content/docus#792] [Q&A] [answered]
First of all, I am really in love with Docus! I like how the devs heard the community and built in many features, like the awesome navigation system.

In the config, we can set the fluid for header, main and footer. It seems, it works for header but not for main. I would like to see the navigation all to the left, the TOC all to the right and the acutal content should take all available space in width. 

Is this a bug or what is the correct setting for that?

https://docus.dev/introduction/configuration

**Comment by lucpotage** [comment, marked as answer]
Hi @derHodrig,

It works on my side with the latest version (v1.9.9) of the theme:

```ts
//app.config.ts
export default defineAppConfig({
  docus: {
    main: {
      fluid: true,
    }
  }
})

```

Reference to the docs: https://docus.dev/introduction/configuration

**Reply by skoenfaelt** [reply]
I was on wrong version
